content/analyst/sfmc/competitive-tracker-installation-guide.md

```diff
@@ -0,0 +1,62 @@
+---
+lastUpdated: "02/23/2023"
+title: Competitive Tracker Installation Guide
+description: This guide helps Salesforce Marketing Cloud admins install SparkPost Competitive Tracker application through the AppExchange
+---
+
+If you are a Salesforce Marketing Cloud Administrator, please follow these one-time steps to install the SparkPost Competitive Tracker application through the AppExchange.
+
+### Step 1. Navigate to the AppExchange installation page by clicking [here](https://mc.exacttarget.com/cloud/#app/Administration/InstalledPackages/8283ba88-2804-4250-bf1d-e1e2b643747a/install)
+
+* You will be automatically be redirected to the installation page if you are logged into SFMC
+
+* In the top right corner, select *Install*
+
+### Step 2. Confirm the installation
+
+ Please do not skip this step! The data extension will not be visible if it is skipped.
+
+* Refresh your browser window
+* Click on the AppExchange icon to make sure that SparkPost Competitive Tracker is a selectable option and open it
+
+![](media/competitive_tracker_installation_guide/image_0.png)
```

Comment:
This image is probably fine but it shows selecting `Sparkpost Inbox Tracker` which might be confusing